lib/id3c/cli/command/etl/manifest.py

```diff
@@ -95,29 +97,40 @@ def etl_manifest(*, db: DatabaseSession):
         with db.savepoint(f"manifest record {manifest_record.id}"):
             LOG.info(f"Processing record {manifest_record.id}")
 
-            # Convert sample barcode to full identifier, ensuring it's
-            # known and from the correct identifier set.
-            sample_barcode = manifest_record.document.pop("sample")
-            sample_identifier = find_identifier(db, sample_barcode)
-
-            if not sample_identifier:
-                LOG.warning(f"Skipping sample with unknown sample barcode «{sample_barcode}»")
+            manifest_type = None
+            try:
+                manifest_type = receiving_manifest.ManifestType(manifest_record.document['manifest_type'])
+            except:
```

Comment:
I would caution against using a general `except` here and narrow it to the Exception you'd expect to be thrown.

Reply:
Absolutely. I'm catching specific exceptions now. Thanks for the call out.

Reply:
The implementation now is quite different from before, so I'm resolving this comment thread. I've caught specific exception types in the new code, but please comment if you see a spot that isn't what you would expect.